### PR TITLE
Doit script

### PR DESCRIPTION
**Pull request by jimafisk**
#### Changes proposed in this pull request:
*  Adds commands to run tests independent of a new build
*  Adds command to open a new MySQL prompt
  * Added less to the db container to allow browsing MySQL via terminal. Adding a second Dockerfile prompted me to restructure how these files are organized (placed them in a dockerfiles directory)
*  Adds command to remove containers and build fresh (helpful if strange behavior is happening in build or you are making changes to underling dockerfiles etc)

### CODE REVIEW

File: doit [hunk 1]
```diff
@@ -110,11 +123,48 @@ function doitbuildhub() {
...
   docker exec -it bostongov_drupal_1 drush rsync @boston.test:%files/ %files -azP
 }
 
+# Open MySQL prompt.
+function doitmysql() {
+  echo $'\r'
+  echo -ne "${Yellow}Opening MySQL promt inside container ${LightGreen}(type \"exit\" or use Ctrl+c to escape).${NC}"
+  echo $'\r'
+  echo $'\r'
+  docker exec -it bostongov_mysql_1 mysql --pager="less -SFX"
```
Comment (finneganh) [inline doit]: 🎉

File: doit [hunk 2]
```diff
@@ -110,11 +123,48 @@ function doitbuildhub() {
...
+}
+
+# Delete all containers and rebuild.
+function doitstartover() {
+  echo $'\r'
+  echo -ne "${Yellow}Stopping any running docker-compose containers.${NC}"
+  echo $'\r'
+  echo $'\r'
+  /usr/local/bin/docker-compose stop
```
Comment (finneganh) [inline doit]: Is the `/usr/local/bin` bit necessary? I’m worried about it being installed somewhere else. We should be confident that it’s on `$PATH`

File: dockerfiles/mysql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM mysql:5.6
+
+WORKDIR /mysql
+RUN apt-get update
+RUN apt-get install less
```
Comment (finneganh) [inline dockerfiles/mysql]: Maybe just drop a comment that this is here so we can use `mysql` in the container with a useful pager.

File: dockerfiles/drupal [hunk 4]
```diff
@@ -8,7 +8,7 @@ RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-av
 
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends git zip unzip bzip2 libbz2-dev rsync nodejs libgd-dev mysql-client openssh-client && \
+    apt-get install -y --no-install-recommends git zip unzip bzip2 libbz2-dev rsync nodejs libgd-dev mysql-client openssh-client vim && \
```
Comment (finneganh) [inline dockerfiles/drupal]: `vim` necessary?

**Comment by finneganh** [comment]
Whoops, should have opened a review to collect all of those comment. Finished pass.

Also squash down to one commit plz.

File: doit (hunk 2 above)
Comment (jimafisk) [reply doit]: Good point, I was worried about that too. Not sure why it wasn't being picked up in my path previously, but seems to work fine now without the absolute path. Fixing this now.

File: dockerfiles/mysql (hunk 3 above)
Comment (jimafisk) [reply dockerfiles/mysql]: Yes definitely, done. Thanks!

File: dockerfiles/drupal (hunk 4 above)
Comment (davidrkupton) [reply dockerfiles/drupal]: Some form of editor is nice to have on a dev container.  When we have got the containers properly configured an editor is arguably not needed -but there is nothing to stop an developer `apt-get install vim` themselves causing their container to have non-standard apps - of course the intention of anyone using vim would be to edit files and therefore make the container have non-standard settings .....
BTW, I needed vim to edit `xdebug` settings, which will need to be on the `bostongov_drupal_1` container if we want to debug in PHPStorm. I'll let @jimafisk know.  I am not sure what @kdonaldson needed to change, but we should note it.
@finneganh

**Comment by jimafisk** [comment]
I got messy with git on this, opening new PR on another branch.

**Pull request closed without merging**